tools/ci_build/github/android/run_nnapi_code_coverage.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Build and run onnxruntime using NNAPI execution provider targeting android emulator
+python3 ~/onnxruntime/tools/ci_build/build.py \
```

Comment:
>~/onnxruntime/tools/ci_build/build.py [](start = 8, length = 37)

This may not be the correct path of the repo, look at nnapi_minimal_build_minimal_ort_and_run_tests.sh to take current directory as a param and get folder from it